slapper.go

```diff
@@ -189,12 +190,15 @@ func (trgt *targeter) readTargets(reader io.Reader, base64body bool) error {
 	return nil
 }
 
-func (trgt *targeter) nextRequest() (*http.Request, error) {
+func (trgt *targeter) nextRequest(randomOrder bool) (*http.Request, error) {
```

Comment:
Hey @JaderDias. Thanks for preparing this PR. I'm surprised that somebody still uses slapper :-)

I see one problem with your approach. Getting a `rand.Int()` involves internal synchronization which may become a bottleneck or a sync point. So, usually, a better way in such cases is to shuffle the original list and still use +1. This way you effectively achieve what you want. For instance, we can prepare a separate targeter for each thread and shuffle the list of requests.

Reply:
Hey @ikruglov thanks for the suggestion. I agree it seems to be the best solution